docs/.sample.env

```diff
@@ -1,2 +0,0 @@
-NOTION_TOKEN=XXXX
-BLOG_INDEX_ID=XXXX
```

Comment:
Can you add the files back? Don’t want to delete these

Reply:
It's done now.